scripts/check_ecosystem.py

```diff
@@ -75,15 +77,26 @@ async def clone(self: Self, checkout_dir: Path) -> "AsyncIterator[Path]":
 
 
 REPOSITORIES = {
-    "airflow": Repository("apache", "airflow", "main", select="ALL"),
-    "bokeh": Repository("bokeh", "bokeh", "branch-3.2", select="ALL"),
-    "build": Repository("pypa", "build", "main"),
-    "cibuildwheel": Repository("pypa", "cibuildwheel", "main"),
-    "disnake": Repository("DisnakeDev", "disnake", "master"),
-    "scikit-build": Repository("scikit-build", "scikit-build", "main"),
-    "scikit-build-core": Repository("scikit-build", "scikit-build-core", "main"),
-    "typeshed": Repository("python", "typeshed", "main", select="PYI"),
-    "zulip": Repository("zulip", "zulip", "main", select="ALL"),
+    "airflow": Repository("apache", "airflow", "main", select="ALL", show_fixes=False),
```

Comment:
How did you decide whether to enable `show_fixes` for a repository?

Reply:
I wanted to do all but it generates a lot of output and bokeh exploded (#4194 and fix convergence failures that i need to write down), so now it's the not-select-ALL cases